server/src/internalClusterTest/java/org/elasticsearch/index/LookupIndexModeIT.java

```diff
@@ -152,12 +151,15 @@ public void testResizeLookupIndex() {
         assertThat(settings.get("index.mode"), equalTo("lookup"));
         assertThat(settings.get("index.number_of_shards"), equalTo("1"));
 
-        ResizeRequest split = new ResizeRequest("lookup-3", "lookup-1");
-        split.setResizeType(ResizeType.SPLIT);
-        split.getTargetIndexRequest().settings(Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 3));
+        ResizeRequest split = ResizeIndexTestUtils.resizeRequest(
+            ResizeType.SPLIT,
+            "lookup-1",
+            "lookup-3",
+            Settings.builder().put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 3)
+        );
         IllegalArgumentException error = expectThrows(
             IllegalArgumentException.class,
-            () -> client().admin().indices().execute(ResizeAction.INSTANCE, split).actionGet()
+            () -> client().admin().indices().execute(TransportResizeAction.TYPE, split).actionGet()
```

Comment:
Tiny nit for consistency:

```suggestion
            () -> client().execute(TransportResizeAction.TYPE, split).actionGet()
```

Reply:
++ b31630c7dfbcf87b74aaab16226c0c83200a63a6